tests/testData.h

```diff
@@ -17,6 +17,44 @@
 
 namespace UnitTest
 {
+
+// Flags that can modify how a test is setUp.  This can be useful
+// for masking tests that are known to be failing, but cannot be
+// resolved at this juncture.  The indices must be unique powers
+// of two in order for masks to be composable.
+//
+// Eventually, once we move to C++23, this can be replaced by a
+// std::bitset, but bitset isn't constexpr until then.
+enum TestFlags
+{
+    TomlFailure = 1, // tests where the TOML testing is known to fail
+};
+
+void compare_toml(std::string location, SerialisedValue toml, SerialisedValue toml2)
```

Comment:
This Pythonically-named function doesn't conform to the style of the rest of the code, so suggest changing to:
```suggestion
void compareToml(std::string location, SerialisedValue toml, SerialisedValue toml2)
```